libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/drillDown.spec.ts

```diff
@@ -136,4 +128,83 @@ describe("Drilling", { tags: ["post-merge_integrated_bear"] }, () => {
             drillModal.getTable().getColumnValues(0).should("deep.equal", ["2011"]);
         });
     });
+
+    describe("Basic drill down", { tags: ["checklist_integrated_bear"] }, () => {
+        it("Should drill down on table with one drillable", () => {
+            Navigation.visit("dashboard/dashboard-table-drill-down");
+            tableData.forEach((tableValue, index) => {
```

Comment:
should change tableValue to insight
tableValue means different meaning

Reply:
Changed tableValue to insight.